site/content/background.mdx

```diff
@@ -0,0 +1,10 @@
+---
+description: Background and origin story of the project
+title: Background
+---
+
+ComposeUI is a dotnet core based, WebView2 using UI Container for hybrid web-desktop applications.
```

Comment:
I'd leave out 'core', just say .NET based

Reply:
Should we say .NET instead of dotnet?

Reply:
Changed dotnet core to .NET